File: sign/sign-interface/src/main/java/com/drzyl/sign/query/SessionQuery.java
package com.drzyl.sign.query;

import java.io.Serializable;
import java.util.List;

import lombok.Data;

/**
 * 会话 SYS_SESSION 查询类
 */
@Data
public class SessionQuery implements Serializable {

  private static final long serialVersionUID = 1L;

  protected String toolExists; // 解决多对多关系预留字段之存在

  protected String toolNotExists; // 解决多对多关系预留字段之不存在

  protected String sessionId; // 会话编号

  protected String accountId; // 账号编号

  protected String loginKey; // 登录标识

  protected Long currentTimes; // 当前时间

  protected String aesKey; // AES密钥

  protected Long cookiesExpireTimesBrowser; // 浏览器会话过期时间

  protected Long cookiesExpireTimesAndroid; // 安卓APP会话过期时间

  protected Long cookiesExpireTimesIos; // 苹果APP会话过期时间

  protected Long cookiesExpireTimesWechatapp; // 微信小程序会话过期时间

  protected List<String> sessionIdAndin; // 会话编号Andin查询

  protected String accountIdAndeq; // 账号编号Andeq查询

  protected String loginKeyAndeq; // 登录标识Andeq查询

  protected String accountIdAndKeyLike; // 账号编号AndKeyLike查询

  protected String loginKeyOrKeyLike; // 登录标识OrKeyLike查询

  protected String aesKeyOrKeyLike; // AES密钥OrKeyLike查询

}

File: vue-coalsale/src/view/com/drzyl/sign/session/SessionQuery.js
export default class SessionQuery {
  constructor () {
    this.toolExists = null // 解决多对多关系预留字段之存在
    this.toolNotExists = null // 解决多对多关系预留字段之不存在
    this.sessionId = null // 会话编号
    this.accountId = null // 账号编号
    this.loginKey = null // 登录标识
    this.currentTimes = null // 当前时间
    this.aesKey = null // AES密钥
    this.cookiesExpireTimesBrowser = null // 浏览器会话过期时间
    this.cookiesExpireTimesAndroid = null // 安卓APP会话过期时间
    this.cookiesExpireTimesIos = null // 苹果APP会话过期时间
    this.cookiesExpireTimesWechatapp = null // 微信小程序会话过期时间
    this.sessionIdAndin = null // 会话编号Andin查询
    this.accountIdAndeq = null // 账号编号Andeq查询
    this.loginKeyAndeq = null // 登录标识Andeq查询
    this.accountIdAndKeyLike = null // 账号编号AndKeyLike查询
    this.loginKeyOrKeyLike = null // 登录标识OrKeyLike查询
    this.aesKeyOrKeyLike = null // AES密钥OrKeyLike查询
  }
}
